src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+
+export const CommentInfo = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
```

Comment:
```suggestion
  <div className="CommentInfo">
```
React fragment is redundant here as you already have a container